shacl12-core/index.html

```diff
@@ -562,14 +648,29 @@ <h3>Document Conventions</h3>
 						<td><code>http://example.com/ns#</code></td>
 					</tr>
 				</table>
+
+				<p>
+					Within this document, the following JSON-LD context is used:
+				</p>
+				<pre class="jsonld">{
+  "@context": {
```

Comment:
Will there be more context added to make the JSON-LD more plain JSON-like?

(It may become impossible to have a better `@context` because of node expressions everywhere.)